src/test/java/seedu/address/logic/parser/task/SearchTaskCommandParserTest.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.parser.task;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.NewCommandTestUtil.DESCRIPTION_DESC_REMIND_PATIENT;
+import static seedu.address.logic.commands.NewCommandTestUtil.INVALID_DATETIME_DUE_FORMAT_DESC;
+import static seedu.address.logic.commands.NewCommandTestUtil.INVALID_DATETIME_DUE_VALUE_DESC;
+import static seedu.address.logic.commands.NewCommandTestUtil.ROOM_NUMBER_DESC_ONE;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.task.SearchTaskCommand;
+import seedu.address.model.task.DateTimeDue;
+
+public class SearchTaskCommandParserTest {
```

Comment:
Can consider adding some success test cases and check if the output tasks are as intended.

Reply:
I will add more success test case in v1.3